src/atc_api/main.go

```diff
@@ -32,9 +33,13 @@ func main() {
 		api.Log.Println("Connected to atcd socket on", args.ThriftAddr)
 	}
 
+	if args.IPv4 == "" || args.IPv6 == "" {
+		api.Log.Fatalln("You must provide both -4 and -6 arguments to run the API.")
```

Comment:
Well, sorry, the wording is correct, but not the behaviour. We want to change this to either one of IPv4/v6 or both right?